HW_7/task34.py

```diff
@@ -0,0 +1,30 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает.
+# Вам стоит написать программу. 
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, 
+# если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+# Пример:
+# Ввод: пара-ра-рам рам-пам-папам па-ра-па-да    
+# Вывод: Парам пам-пам 
+
+rhyme = input('Введите "рифму": ').split(" ")
```

Comment:
Тут можно сразу же привести входную строку к одному регистру. Тогда не придется использовать переменную `dictionary` таким образом. И, если вам нужно разделить стhоку по пробелаv, то лучше ничего не передавать методу `split`
```suggestion
rhyme = input('Введите "рифму": ').lower().split()
```